src/components/voucher/VoucherPlanDetailModal.tsx

```diff
@@ -102,12 +131,67 @@ const VoucherPlanDetailBlock: React.FC<{ title: string; voucherPlanId: string }>
               isFinished={voucherCode.used === voucherCode.count}
             >
               <code>{voucherCode.code}</code>
-              <div>
+              <StyledVoucherCodeAmountBlock>
                 {formatMessage(promotionMessages.text.exchangedCount, {
                   exchanged: voucherCode.count - voucherCode.remaining,
                   total: voucherCode.count,
                 })}
-              </div>
+                <AdminModal
+                  renderTrigger={({ setVisible }) => (
+                    <DeleteOutlined className="ml-4" onClick={() => setVisible(true)} />
+                  )}
+                  title={formatMessage(
+                    voucherCode.vouchers.some(voucher => voucher.used === true)
```

Comment:
`voucherCode.vouchers.some(voucher => voucher.used === true)` can be `voucherCode.vouchers.some(voucher => voucher.used)`

used is boolean